packages/new-design/templates/shared/index.mjs.mustache

```diff
@@ -1,13 +1,17 @@
+{{!
```

Comment:
```suggestion
//{{!
```
as in #2971, and ditto for all the files below

Reply:
> I found that the old one just made the first line of the file be `//`.

Ah, yes, that's right. Per the code comment itself, all of this came out of my efforts to [get our Mustache templates to pass lint](https://github.com/freesewing/freesewing/pull/2970/commits/93788aa0b41f63810b48793b7db0701269827799). In files like this, the best solution I could find was to [change the delimiter](https://github.com/janl/mustache.js#custom-delimiters) with a comment in front of the delimiter change. This does unfortunately produce an empty comment in the Mustache output.

(I think I'd previously tried to fix all our Mustache linting problems by writing an [eslint preprocessor](https://eslint.org/docs/latest/developer-guide/working-with-plugins#processors-in-plugins) that could render Mustache templates, but IIRC it's nontrivial to find dummy values for the various template variables. In any case I guess it's probably a good thing for our JS files to be syntactically correct, for the sake of prettier and editors and other static analysis tools.)

> I imagine we'll need to fix the Mustache templates, at the very least

Oops, disregard this, it's lower-priority than I'd feared. Because you're right, syntax issues are cleaned up by Mustache before the file ever reaches a JS interpreter. I can simply follow up on this in my next linting PR.

Reply:
Probably the thing for us to do is  add the mustache files to the lint ignore, then make a `prelint` script that generates the files for from scratch, tutorial, and from bent. Then those will be linted and a `postlint` can clean then up